tests/unit/test_chainer.py

```diff
@@ -248,6 +248,32 @@ def test_create_model(sagemaker_session, chainer_version):
     assert model.enable_cloudwatch_metrics == enable_cloudwatch_metrics
 
 
+def test_create_model_with_custom_image(sagemaker_session):
+    container_log_level = '"logging.INFO"'
+    source_dir = 's3://mybucket/source'
+    enable_cloudwatch_metrics = 'true'
+    custom_image = 'ubuntu:latest'
+    chainer = Chainer(entry_point=SCRIPT_PATH, role=ROLE, sagemaker_session=sagemaker_session,
+                      train_instance_count=INSTANCE_COUNT, train_instance_type=INSTANCE_TYPE,
+                      image_name=custom_image, container_log_level=container_log_level,
+                      py_version=PYTHON_VERSION, base_job_name='job', source_dir=source_dir,
+                      enable_cloudwatch_metrics=enable_cloudwatch_metrics)
+
+    job_name = 'new_name'
+    chainer.fit(inputs='s3://mybucket/train', job_name='new_name')
+    model = chainer.create_model()
+    chainer.container_log_level
+
+    assert model.sagemaker_session == sagemaker_session
+    assert model.image == custom_image
```

Comment:
I think I would keep the asserts minimal for this test, and just assert the image, assuming that other unit tests cover the other parameters already.